src/test/scala/com/gd/twitteranalytics/TweetsDataLoaderTest.scala

```diff
@@ -0,0 +1,47 @@
+package com.gd.twitteranalytics
+
+import java.io.File
+import java.util.{Date => JavaDate}
+import java.sql.Date
+
+import com.gd.twitteranalytics.util.AppConfigReader
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.scalatest.FreeSpec
+
+class TweetsDataLoaderTest extends FreeSpec with DataFrameSuiteBase {
+
+  "TweetsDataLoader" - {
```

Comment:
all data that was generated in results of testing should be deleted from an File System.

Reply:
Done !